tests/devices/test_default_qutrit.py

```diff
@@ -72,6 +72,152 @@ class TestApply:
     """
 
     # TODO: Add tests for non-parametric ops after they're implemented
+    test_data_no_parameters = [
+        (qml.TShift, [1, 0, 0], np.array([0, 1, 0]), None),
+        (
+            qml.TShift,
+            [1 / math.sqrt(2), 1 / math.sqrt(2), 0],
+            np.array([0, 1 / math.sqrt(2), 1 / math.sqrt(2)]),
+            None,
+        ),
+        (qml.TClock, [1, 0, 0], np.array([1, 0, 0]), None),
+        (qml.TClock, [0, 1, 0], np.array([0, OMEGA, 0]), None),
+    ]
+
+    test_data_no_parameters_inverses = [
+        (qml.TShift, [0, 1, 0], np.array([1, 0, 0]), None),
+        (
+            qml.TShift,
+            [0, 1 / math.sqrt(2), 1 / math.sqrt(2)],
+            np.array([1 / math.sqrt(2), 1 / math.sqrt(2), 0]),
+            None,
+        ),
+        (qml.TClock, [1, 0, 0], np.array([1, 0, 0]), None),
+        (qml.TClock, [0, OMEGA, 0], np.array([0, 1, 0]), None),
+    ]
+
+    @pytest.mark.parametrize("operation, input, expected_output, subspace", test_data_no_parameters)
```

Comment:
Just to clarify - subspace is here because it will be needed in future tests with the subspace ops, correct?

Reply:
That's correct.